cleverpush/src/main/java/com/cleverpush/stories/StoryDetailActivity.java

```diff
@@ -13,20 +13,23 @@
 import androidx.recyclerview.widget.SnapHelper;
 
 import com.cleverpush.R;
-import com.cleverpush.stories.Utils.OnSwipeTouchListener;
+import com.cleverpush.stories.listener.OnSwipeDownListener;
+import com.cleverpush.stories.listener.OnSwipeTouchListener;
+import com.cleverpush.stories.listener.StoryChangeListener;
 import com.cleverpush.stories.models.Story;
 
 import java.util.ArrayList;
 
 
-public class StoryDetailActivity extends Activity {
+public class StoryDetailActivity extends Activity implements StoryChangeListener {
 
     StoryDetailListAdapter storyDetailListAdapter;
-    private ArrayList<Story> stories = new ArrayList<>();
-    private int selectedPosition = 0;
     RecyclerView recyclerView;
```

Comment:
is there a reason there are no modifiers to them? are they supposed to be package-scope only?